consensus/src/checkpoint.rs

```diff
@@ -0,0 +1,142 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::common::{Common, ProtocolStateChain};
+use mina_crypto::base58::Base58Encodable;
+use mina_crypto::hash::{EpochSeed, StateHash};
+use mina_rs_base::protocol_state::ProtocolState;
+use wasm_bindgen_test::*;
+const SLOTS_PER_EPOCH: u32 = 7140;
+
+/// init_checkpoints initializes the checkpoints for the genesis block
+/// This function assumes the state hash of `genesis` is already set
```

Comment:
can remove this comment, as we do initialize the two checkpoints within this function.